go.mod

```diff
@@ -130,12 +130,36 @@ require (
 )
 
 exclude (
+	github.com/containerd/containerd v1.2.10
+	// exclude github.com/containerd/containerd < 1.6.1, 1.5.10, 1.14.12 https://nvd.nist.gov/vuln/detail/CVE-2022-23648
+	github.com/containerd/containerd v1.2.7
+	github.com/containerd/containerd v1.3.0
+	github.com/containerd/containerd v1.3.2
+	github.com/containerd/containerd v1.4.1
+	github.com/containerd/containerd v1.4.3
+	github.com/containerd/containerd v1.4.4
+	github.com/containerd/containerd v1.4.9
+	github.com/containerd/containerd v1.5.0-beta.1
+	github.com/containerd/containerd v1.5.0-beta.3
+	github.com/containerd/containerd v1.5.0-beta.4
+	github.com/containerd/containerd v1.5.0-rc.0
+	github.com/containerd/containerd v1.5.1
+	github.com/containerd/containerd v1.5.2
+	github.com/containerd/containerd v1.5.7
+	github.com/containerd/containerd v1.5.9
 	// force use of go.etcd.io/bbolt
 	github.com/coreos/bbolt v1.3.0
 	github.com/coreos/bbolt v1.3.2
 	github.com/coreos/bbolt v1.3.3
 	github.com/etcd-io/bbolt v1.3.3
 	github.com/etcd-io/bbolt v1.3.6
+	// exclude github.com/golang/protobuf < 1.3.2 https://nvd.nist.gov/vuln/detail/CVE-2021-3121
+	github.com/gogo/protobuf v1.0.0
+	github.com/gogo/protobuf v1.1.1
+	github.com/gogo/protobuf v1.2.0
+	github.com/gogo/protobuf v1.2.1
+	github.com/gogo/protobuf v1.3.0
+	github.com/gogo/protobuf v1.3.1
```

Comment:
See [this comment](https://github.com/Azure/ARO-RP/pull/1992#discussion_r818533904) form another PR:

> There are more [tags](https://github.com/gogo/protobuf/tags): do we need to exclude rest of them? E.g. `v0.5`, `v0.4`, etc